mk/android.mk

```diff
@@ -47,6 +53,7 @@ ONELIB=0
 OSTYPE=android
 LINK=
 #CC_AR=ndk-ar -r ${LIBAR}
+CC_AR=arm-linux-androideaby-ar -r ${LIBAR}
```

Comment:
uh. that should be only for dockcross? it shuoldbe fine for the sdk i guess

also.. i doubt androideabY exists. it's 'i'